### ISSUE TITLE
sess.run() in tensorflow2.1

### ISSUE THREAD

**Issue by DaerTaeKook** [tensorflow/tensorflow#40123]
<em>Please make sure that this is a bug. As per our
[GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md),
we only address code/doc bugs, performance issues, feature requests and
build/installation issues on GitHub. tag:bug_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 18.04):
- TensorFlow installed from (pip source form Tsinghua mirror):
- TensorFlow version (2.1):
- Python version:3.6.7
- GPU model and memory:without GPU

**Describe the current behavior**
when I used such code:
with tf.Graph().as_default() as graph:
    features = input_fn(input_fn_params)
    model_fn(features=features,
             labels=None,
             mode=tf.estimator.ModeKeys.PREDICT,
             params=model_fn_params)

    tf.train.Saver()
    graph_def = graph.as_graph_def(add_shapes=True)
    tf.train.write_graph(graph_def, directory, 'inference.pbtxt')
    meta_graph_name = os.path.join(directory, 'inference.meta')
    tf.train.export_meta_graph(filename=meta_graph_name)

How can I transform tensor to ndarray in  tf.Graph().as_default()
